connector/src/main/java/org/jboss/as/connector/logging/ConnectorLogger.java

```diff
@@ -778,13 +778,13 @@
     DeploymentUnitProcessingException invalidConnectionFactory(String cf, String ra, String jndiName);
 
     /**
-     * Admin object declared for JCA 1.0 archive
+     * Admin object declared for Jakarta Connectors archive
      *
      * @param ra       The resource adapter
      * @param jndiName The JNDI name
      * @return a {@link DeploymentUnitProcessingException} for the error.
      */
-    @Message(id = 84, value = "Admin object declared for JCA 1.0 resource adapter '%s' while deploying %s")
+    @Message(id = 84, value = "Admin object declared for Jakarta Connectors resource adapter '%s' while deploying %s")
```

Comment:
I'm unsure about this change, this is a feature of a specific version of JCA spec, I think we should keep it as is